control/strategy/pool.go

```diff
@@ -297,6 +298,20 @@ func (p *pool) Kill(id uint32, reason string) {
 	}
 }
 
+// Kill all instances of a plugin
+func (p *pool) KillAll(reason string) {
+	for _, rp := range p.plugins {
+		rp.Stop(reason)
```

Comment:
Looking at it again can we just re-use the Kill() function? It takes the right inputs and I think it fits that KillAll will just look through and call kill on each plugin individually.